readme.md

```diff
@@ -139,8 +181,14 @@ Define a Javascript constructor called 'Instructor'. Every instance of Instructo
 Instantiate an instructor named 'Andy' and call its `receivePresent` method with "Resin Laying Deer Figurine, Gold" as the argument.
 
 ```js
-// Your code starts here...
-
+var Instructor = function(name){
+  this.name = name;
+  this.receivePresent = function(gift){
+    console.log(name+"promptly drops the "+gift+" on the floor.");
+  }
+}
+Andy = new Instructor("Andy");
+Andy.receivePresent("Resin Laying Deer Figurine, Gold")
 ```
```

Comment:
this is fine and works great. I would however define receivePresent as a prototyped method instead of defining it on this in the constructor. Every time we create a new instructor we'll also be creating that function as a property of the object in memory. If we prototype it, its only defined once.